HelpSource/Classes/Phasor.schelp

```diff
@@ -58,6 +58,21 @@ Examples::
 
 code::
 
+// basic: ramp between 'start' and 'end'
+// note that the ramp ends at 9, not 10
+{ Phasor.kr(0, 1, start: 0, end: 10) }.plot(0.03)
```

Comment:
With examples like this, it can be helpful to make use of `plotMode_`, in particular `\dlines`, to illustrate increment sizes, etc.

`{ Phasor.kr(0, 1, start: 0, end: 10) }.plot(0.03).plotMode_(\dlines)`